core/Test/Tasty/Patterns.hs

```diff
@@ -44,7 +47,9 @@ instance IsOption TestPattern where
   parseValue = parseTestPattern
   optionName = return "pattern"
   optionHelp = return "Select only tests which satisfy a pattern or awk expression"
-  optionCLParser = mkOptionCLParser (short 'p' <> metavar "PATTERN")
+  optionCLParser =
```

Comment:
@rhendric could you please attach a comment to the `instance IsOption TestPattern` saying "Since tasty-1.5 this command-line option can be specified multiple times with accumulating effect bla-bla-bla" or similar?

Reply:
Done.